packages/web-app-files/src/components/SideBar/Shares/Collaborators/ListItem.vue

```diff
@@ -13,6 +13,13 @@
         :width="48"
         class="files-collaborators-collaborator-indicator"
       />
+      <oc-avatar-guest
```

Comment:
Was this not working with the `oc-avatar-item` below? That one was intended to reflect the correct data based on the given share type. Which should work in theory 😅

Reply:
As mentioned before in chat: 
:icon="shareTypeKey" would be "guest",  but we want to have the icon "global"

Reply:
What about a computed prop `shareTypeIcon` where you return `global` if share type is guest, and in all other cases just the value of `shareTypeKey`? then you can keep the template clean and only have a different case for icon selection.

Reply:
I have the feeling, that adding the icon to the share type, would be much cleaner and future proof. 
Do you disagree?